pom.xml

```diff
@@ -9,17 +9,32 @@
 
     <properties>
         <java.version>1.8</java.version>
-        <handlebars-java-version>4.0.3</handlebars-java-version>
+        <handlebars-java-version>4.3.1</handlebars-java-version>
+        <handlebars-markdown-java-version>4.2.1</handlebars-markdown-java-version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <powermock.version>1.5.6</powermock.version>
-        <dp-logging.version>v2.0.0-beta.8</dp-logging.version>
-        <fasterxml.jackson.version>2.15.0</fasterxml.jackson.version>
+        <dp-logging.version>v2.0.0-beta.10</dp-logging.version>
+        <fasterxml.jackson.version>2.16.1</fasterxml.jackson.version>
         <jetty.version>9.4.53.v20231009</jetty.version>
-        <flying-saucer.version>9.1.22</flying-saucer.version>
+        <flying-saucer.version>9.4.0</flying-saucer.version>
     </properties>
 
     <dependencies>
-
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.3.14</version>
+        </dependency>
+        <dependency>
```

Comment:
Has this dependency being pinned here to fix a vulnerability? If so, you should add a comment on top of this dependency to explain which vulnerability this fixes (so you'll need to add the vulnerability ID/code).

Reply:
This has to be included otherwise the tests will not run. Even it is included by the dp-logging beta.10 it has to be included here again. Its a bit weird.  I thought its a scope issue but default is "compile" which includes "runtime".

Reply:
It looks like it's being used by `RequestDelegator`, but it was never imported explicitly in the pom, which means that this was a transitive dependency that was removed after your updates. So adding it here like you've done is the right thing to do 👍